tasks/docker.rb

```diff
@@ -91,7 +91,14 @@ def provision(docker_platform, inventory_location)
     break unless stdout.include?(ports)
     raise 'All front facing ports are in use.' if front_facing_port == 2230
   end
-  creation_command = "docker run -d -it --privileged -p #{front_facing_port}:22 --name #{full_container_name} #{docker_platform}"
+  # setting volume parameter for debian9 and ubuntu16/18
+  os_string = docker_platform.split('/').last
+  if os_string == 'debian9' || os_string == 'ubuntu16.04' || os_string == 'ubuntu18.04'
+    deb_family_systemd_volume = '--volume /sys/fs/cgroup:/sys/fs/cgroup:ro'
+  else
+    deb_family_systemd_volume = ""
+  end
```

Comment:
`deb_family_systemd_volume = if dockerplatform ~= r%{debian|ubuntu} and   dockerplatform  ~! r{debian8|ubuntu14} 
                                                       '--volume /sys/fs/cgroup:/sys/fs/cgroup:ro'
                                                      else
                                                       ''
                                                       end

`